configs/Gigabyte/GA-M68M-S2P.conf

```diff
@@ -0,0 +1,87 @@
+# Configuration file for the Gigabyte GA-M68M-SP2
+# Contributed by Clipo.
+# 27-12-2019 
+# Updates welcome.
+
+# CPU sensor
+
+chip "k8temp-*"
+
+    label temp1 "Core0 Temp"
+    label temp2 "Core0 Temp"
+    label temp3 "Core1 Temp"
+    label temp4 "Core1 Temp"
+
+# Motherboard sensors
+
+chip "it8718-*"
+
+### Voltages
+
+    label  in0  "Vcore"     # label matched to BIOS
+    label  in1  "DDR2 1.8V" # label matched to BIOS
+    label  in2  "+3.3V"     # label matched to BIOS
+    ignore  in3             # None changing value of 4.08 assumed not used
+    label  in4  "+12V"      # label matched to BIOS
+    ignore  in5             # None changing value of 4.08 assumed not used    
+    ignore  in6             # None changing value of 4.08 assumed not used
+    label  in7  "+5VSB"      # Not shown in BIOS
+    label  in8  "VBat"
+    
+# "Vcore", "DDR2 1.8V", "+3.3V" and "Vbat" are connected directly, so no compute
+# line is needed for these.
+   
+#   compute in3  ((6.8/10)+1)*@ , @/((6.8/10)+1) # 5V standard compute
+    compute in4  ((30/10)+1)*@  , @/((30/10)+1)
+#   compute in5  (1+232/56)*@ - 4.096*232/56 , (@ + 4.096*232/56)/(1+232/56)
+#   compute in6  (1+120/56)*@ - 4.096*120/56 , (@ + 4.096*120/56)/(1+120/56)
+    compute in7 ((6.8/10)+1)*@ , @/((6.8/10)+1)
+
+# 
+   # The BIOS will not set any limit for voltages.
+
+   set in0_min  1.35  * 0.95 # Vcore 1.35V - 1.4V
+   set in0_max  1.4   * 1.05 # Vcore 1.35V - 1.4V
+   set in1_min  1.8   * 0.95 # DDR2 1.8V
+   set in1_max  1.8   * 1.05 # DDR2 1.8V
+   set in2_min  3.3   * 0.95 # 3.3V
+   set in2_max  3.3   * 1.05 # 3.3V
+   set in3_min  5     * 0.95 # Not used
+   set in3_max  5     * 1.05 # Not used
+   set in4_min 12     * 0.95 # 12V
+   set in4_max 12     * 1.05 # 12V
+   set in7_min  5     * 0.95 # 5V
+   set in7_max  5     * 1.05 # 5V
+
+### Temperatures
+
+   label temp1 "System Temp"
+   # This sensor appears to be around or inside the ITE8718 chip.
+   set   temp1_min 0
+   set   temp1_max 60 # Set to the to match your BIOS "System Warning Temperature" 
+
+   label temp3  "CPU Temp"
+   # This sensor can show 2 degrees difference to the K8 sensor.
+   # The end user can choose which one they prefer.
+   
+   set   temp3_min 0
+   set   temp3_max 60 # Set to match your BIOS "CPU Warning Temperature"
+
+   label  temp2  "NB Temp" # Assumed to be North Bridge as temerature follows load
+      
+### Fans
+
+   # Label based on motherboard connectors.
+   label fan1 "CPU Fan"		# M/B label: CPU_FAN (4-pin)
+   label fan2 "System Fan"	# M/B label: SYS_FAN (3-pin)
+   
+   # Minimum fan speeds.
+   # Comment/un-comment fan headers you are using if you want alarms.
+   set fan1_min 1000	# CPU Fan
+   set fan2_min 1000	# System Fan
+
+### Ignore statements
+
+   ignore cpu0_vid # None changing value from 1.4V
```

Comment:
If I understand VID correctly, then a non-changing value is valid here. See the file `doc/vid`. I think this line should be removed.